tests/config_parser_test.cc

```diff
@@ -0,0 +1,247 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+#include "gtest/gtest.h"
+#include "gmock/gmock.h"
+
+#include "example/usps_api/config/config_parser.h"
+#include "proto/usps_api/ghost_label.pb.h"
+#include <string>
+#include <fstream>
+#include <list>
+#include <iostream>
+#include <cstdio>
+#include "json/json.h"
+
+namespace {
+  std::string filename = "example/usps_api/config/config.json";
```

Comment:
See the section on naming constant variables in the style guide.

Reply:
fixed!